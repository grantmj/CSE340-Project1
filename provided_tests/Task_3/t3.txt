TASKS
    1 3

POLY
    F(x,y) = x^2 x^3 y + 2;

EXECUTE
    INPUT A;
    H = F(A,B);

INPUTS
    3 4 5
