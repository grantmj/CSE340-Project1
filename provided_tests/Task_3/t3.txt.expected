POLY - SORTED MONOMIAL LISTS
	F(x,y) = x^5 y + 2;
